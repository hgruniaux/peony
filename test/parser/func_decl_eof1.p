//# RUN: peony %s

fn foo(
//~^ ERROR: expected parameter declarator
//~| ERROR: expected ')', found 'EOF'
