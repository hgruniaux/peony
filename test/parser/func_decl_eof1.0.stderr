<source>:3:8: error: expected parameter declarator
    3 | fn foo(
      |        ^
<source>:3:8: error: expected ')', found 'EOF'
